.cirrus.yml

```diff
@@ -155,6 +155,17 @@ meta_task:
     clone_script: &noop mkdir -p $CIRRUS_WORKING_DIR  # source not needed
     script: /usr/local/bin/entrypoint.sh
 
+fedora36_build_task:
+    alias: Fedora_36_rust_build
+    depends_on:
+      - "build"
+    container:
+        cpu: 2
+        memory: 2
+        image: quay.io/libpod/fedora36rust
```

Comment:
what is to be nervous about?  this is supposed to mimic actual distribution builds so we don't incur problems when it comes to packaging.  should it not be latest? that is the one that gets update.  tell me more

Reply:
>that is the one that gets update

And that's the crux. Said updates are out-of-band wrt the code that relies on it (here in CI). We've seen this happen several times: `foo.image:latest` gets updated, and a few days later something breaks on a maintenance branch because that update changed a behavior that nobody even knew the old branch relied on. See, e.g., https://github.com/containers/podman/pull/12343

Reply:
I think we actually want that in the case though, no?  If the distribution ships an updated cargo, then we want to be testing with that on all branches, as we are pegged to the distribution level here; not the cargo version.  What so you?

Reply:
It's a tradeoff. It's one that we make often, and sometimes it's the right call, sometimes it's not. 

I'm not blocking the PR, merely calling attention to it. If this is a deliberate and well-thought-out choice, then so be it. But could I at least request that you give the existing image a secondary descriptive tag? That way if/when there's a SNAFU with an image upgrade, we can do an emergency revert.